dashboard/model_layer.py

```diff
@@ -128,14 +128,14 @@ def get_node_time_online(day):
     node.country_string = get_country_string(
         node.get_country_from_service_proposal()
     )
-    node.sessions = get_sessions(node_key=node_key)
+    node.sessions = get_sessions(node_key=node_key, limit=10)
```

Comment:
Maybe it should be named `node.last_sessions` to clarify that not all sessions are here?